practice-one/src/styles/components/special-links.css

```diff
@@ -0,0 +1,5 @@
+.special-link {
```

Comment:
Naming conventions is not good. You can use btn class for the <a> tag. You can refer the bootstrap the define the class name https://getbootstrap.com/docs/5.0/components/buttons/